$NetBSD: patch-meson.build,v 1.4 2024/01/30 13:54:32 ryoon Exp $

* Non-MS Windows case uses libintl for dgettext.
  Fix build under NetBSD.

--- meson.build.orig	2022-05-28 15:18:37.000000000 +0000
+++ meson.build
@@ -368,7 +368,7 @@ if cc.has_header_symbol('sys/syscall.h',
 endif
 
 if cc.has_function('dgettext')
-  if host_machine.system() != 'windows'
+  if host_machine.system() == 'windows'
     libintl_dep = []
   else
     libintl_dep = cc.find_library('intl')
